test/infamy/multicast.py

```diff
@@ -54,3 +55,29 @@ def __exit__(self, _, __, ___):
                 print("ERR")
                 pass
         self.proc.wait()
+        
+class MacMCastSender:
+    def __init__(self, netns, group):
+        self.group = group
+        self.netns = netns
+
+    def __enter__(self):
+        send_cmd = (
+            "from scapy.all import sendp, Ether; "
+            f"pkt=Ether(src='d2:68:c1:eb:e4:26', dst='{self.group}', type=0xdead); "
```

Comment:
Do you required to specify src? If not it feel strange to have random hard coded address, should it not be the one of the exiting interface?

Reply:
Thanks. It's leftover from debugging. I'll remove it :)

Reply:
Changed to something that does not look like an actual MAC address, since scapy doesn't seem to allow avoiding src addr